nap/docker/ckan/ckan-plugins/ckanext-napote_theme/ckanext/napote_theme/templates/home/layout1.html

```diff
@@ -1,24 +1,27 @@
 <div role="main" class="hero">
     <div class="container">
-        {% if c.userobj %}
-            <script type="text/javascript">
+        <script type="text/javascript">
                 window.addEventListener('load', function hideTestEnvInfo() {
                     var el = window.document.querySelector('#test-env-warning');
 
                     if (el && window.location.hostname.indexOf('testi') !== -1) {
-                        el.style.display = 'block';
+                        el.style = 'border: red 4px dashed;';
```

Comment:
Illogical style definition here.

Reply:
We move that element to -9999 in production but in testing we replace that wrong placement with borders and element is show where it should be.

Reply:
This behaviour could be commented better. Otherwise, ok.